addons/api/addon/generated/models/credential.js

```diff
@@ -54,6 +54,13 @@ export default class GeneratedCredentialModel extends BaseModel {
   })
   password_hmac;
 
+  @attr('string', {
+    for: 'username_password',
+    isNestedAttribute: true,
```

Comment:
Just want to confirm, are these nested under `attributes`?  e.g. `attributes.password`?

Reply:
Yes. Password, password_hmac, and username are all nested under attributes.